README.md

```diff
@@ -1,7 +1,19 @@
 # Northcoders House of Games API
 
+## Project details
+
+This project is an API for a boardgames review website. It was made as a backend portfolio piece as part of the Northcoders Coding Bootcamp.
+
+A hosted version is available on https://rk-board-games.herokuapp.com/api/
+
 ## Setup Instructions
 
+This repository is available to fork and clone from https://github.com/RossKeen/Boardgame-Backend-Project
+
+Ensure that all dependencies are installed locally by running `npm install` on the command line from within the repository.
```

Comment:
it CAN be nice to write all commands that someone has to use in a codeblock by itself.  If you have many commands in a row you can add comments above/next to them to show what they do